sources/admin/ManageFeatures.controller.php

```diff
@@ -82,6 +82,9 @@ class ManageFeatures_Controller extends Action_Controller
 	public function pre_dispatch()
 	{
 		$this->_req = HttpReq::instance();
+
+		// We need this in few places so it's easier to have it loaded here
+		require_once(SUBSDIR . '/ManageFeatures.subs.php');
```

Comment:
:smile: I was thinking to do the same thing in most of those areas ... call in the base subs for the controller instead of having it in a dozen places !